spec/factories/event_stream.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :event_stream do
+    sequence(:id) { SecureRandom.random_number(100) }
```

Comment:
Not sure I understand the need for this. In particular, overriding the database's internal allocation of ids seems like it could be a bad idea (especially since this is tested against a random region - could lead to unexpected results)